google/cloud/sql/connector/connector.py

```diff
@@ -17,7 +17,55 @@
 import googleapiclient
 
 
-def get_ephemeral(service, proj_name, inst_name, pub_key):
+def get_metadata(service, project, instance):
+    """
+    A helper function that requests metadata from the Cloud SQL Instance
+    and returns a dictionary containing the IP addresses and certificate
+    authority of the Cloud SQL Instance.
+
+    Args:
+        service (googleapiclient.discovery.Resource): A service object created
+          from the Google Python API client library. Must be using the SQL
+          Admin API. For more info check out
+          https://github.com/googleapis/google-api-python-client.
+        proj_name (str): A string representing the name of the project.
+        inst_name(str): A string representing the name of the instance.
+
+    Returns:
+        Dict[str: Dict[str: str], str: str]
```

Comment:
I can't map this to https://www.python.org/dev/peps/pep-0484/ and trying to figure out what this would be.

I think `so Dict[str, Union[Dict, str]]`?